lib/service_actor/failable.rb

```diff
@@ -17,11 +17,11 @@ module ClassMethods
       def inherited(child)
         super
 
-        child.fail_ons.append(*fail_ons)
+        child.fail_ons.push(*fail_ons)
```

Comment:
`append` was added  in 2.5 an is just an alias to `push` so I changed it back for compatibility